environments/docker-compose.local.yml

```diff
@@ -0,0 +1,22 @@
+---
+version: "3.8"
+services:
+  nautobot:
+    command: "nautobot-server runserver 0.0.0.0:8080"
+    ports:
+      - "8080:8080"
+    volumes:
+      - "../config/nautobot_config.py:/opt/nautobot/nautobot_config.py"
+      - "../jobs:/opt/nautobot/jobs"
+    healthcheck:
+      test: ["CMD", "true"]  # Due to layering, disable: true won't work. Instead, change the test
```

Comment:
For a deployment, we shouldn't be disabling the healthcheck like this.

Reply:
Fair. I think we need to come up with a good healthcheck.

Reply:
The docker image provides a healthcheck, shouldn't we just leave it as default?

Reply:
I've removed the healthchecks from the local compose file. However, it looks like the cookiecutter-nautobot-app pattern has one for Postgres so I've copied that in.